website/docs/r/cognito_identity_pool_roles_attachment.markdown

```diff
@@ -119,17 +119,12 @@ The Cognito Identity Pool Roles Attachment argument layout is a structure compos
 
 ## Attributes Reference
 
-In addition to all arguments above, the following attributes are exported:
-
-* `id` - The identity pool ID.
-* `identity_pool_id` (Required) - An identity pool ID in the format REGION:GUID.
-* `role_mapping` (Optional) - The List of [Role Mapping](#role-mappings).
-* `roles` (Required) - The map of roles associated with this pool. For a given role, the key will be either "authenticated" or "unauthenticated" and the value will be the Role ARN.
+No additional attributes are exported.
```

Comment:
I think given that `id` is set to the `identity_pool_id` ([defined here](https://github.com/hashicorp/terraform-provider-aws/blob/78b2dea023f65b5644f783663f32942227e16966/internal/service/cognitoidentity/pool_roles_attachment.go#L139)), it's probably worth keeping the `id` attribute documented. The others are arguments above, and I totally agree with removal.